src/SourceBuild/patches/winforms/0001-Condition-out-the-native-tests-in-VMR.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "Nikola Milosavljevic (CLR) false" <nikolam@microsoft.com>
+Date: Tue, 25 Mar 2025 07:52:19 -0700
+Subject: [PATCH] Condition out the native tests in VMR
+
+Backport: https://github.com/dotnet/winforms/issues/13186
```

Comment:
Reading the issues, it appears that you intend to keep this patch until the native issue is resolved.  This patch should be proactively backported.  It eliminates any work on our side to remove the patch when they fix their issue. Also, I suspect they may not fix their issue right away.  All patches will need to be eliminated in P4 when we switch to the flat flow.

Reply:
Sound good, I would just update the backport link to reference your PR.